backend/README.md

```diff
@@ -153,50 +166,28 @@ If you don't want to use docker alone, you can also set up and run the dev envir
 `psql` is required to load data into the db, but it should be installed when you install `postgres`/`psycopg2`.
 
 1. `cd backend`
-2. Compiling postgres (`psycopg2`)
-
-   - **Mac**
-     > :warning: NOTE: If your computer runs on Apple silicon and you use Rosetta to run Python as an x86 program, use `arch -x86_64 brew <rest of command>` for all `brew` commands.
-
-     1. `brew install postgresql`
-     2. `brew install openssl`
-     3. `brew unlink openssl && brew link openssl --force`
-     4. Follow the instructions printed by the previous command to add openssl to your PATH and export flags for compilers, e.g.:
-        - `echo 'export PATH="/usr/local/opt/openssl@3/bin:$PATH"' >> ~/.zshrc`
-        - `export LDFLAGS="-L/usr/local/opt/openssl@3/lib"`
-        - `export CPPFLAGS="-I/usr/local/opt/openssl@3/include"`
-
-   - **Windows (WSL) or Linux:**
-     - `apt-get install gcc python3-dev libpq-dev`
-
-3. Running Docker
-   1. Open a new terminal window (also in the `backend` directory) and run `docker-compose up`
+2. Compiling postgres (`psycopg2`) and installing dependencies
+   - Run `make install` in the background directory
+3. Running Containerized Database and Cache
+    - Open a new terminal window and run `make launchdb` to start the database and cache
       > :warning: Depending on your system configuration, you may have to start `docker` manually. If this is the case (ie, if you cannot get `docker-compose up` to work due to a docker connection error) try this:
-      >
       > - (linux) `[sudo] systemctl start docker`
       > - (WSL) `[sudo] service docker start`
 
-4. Setting up your Penn Courses development environment
-
-   1. `pipenv install --dev`
-   2. `pipenv shell`
-   3. `python manage.py migrate`
-
-5. Loading test data (if you are a member of Penn Labs). If you are not a member of Penn Labs, you can skip this section and load in course data from the registrar, as explained below.
+4. Migrate Django models
```

Comment:
Hmm, I think using make for simple things like migrate/makemigrations could end up being a leaky abstraction as we will not provide make commands for all the commands they might run (e.g., management commands or manage.py shell). 

IMO, I think we should add make commands for the installation, but otherwise ask people to use `python manage.py xxx`.